status.go

```diff
@@ -264,6 +271,19 @@ func initStatus(services []*service) {
 			http.Error(w, "service not found", http.StatusNotFound)
 			return
 		}
+
+		if jsonRequested(r) {
+			b, err := json.Marshal(svc)
+			if err != nil {
+				http.Error(w, err.Error(), http.StatusInternalServerError)
+				return
+			}
+
+			w.Header().Set("Content-Type", "application/json")
+			_, _ = fmt.Fprint(w, string(b))
```

Comment:
Here and below: don’t use `Fprint` without specifying a format. Instead, just use `w.Write(b)` for writing out JSON.